_gsocproposals/2019/proposal_MLaaS4HEP.md

```diff
@@ -5,7 +5,7 @@ project: CMS
 year: 2019
 organization: 
   - Cornell
-  - Université Clermont Auvergne
+  - CNRS/IN2P3
```

Comment:
Last year Sebastien was affiliated with `LPC-Clermont` - and I think it's still right! Maybe it's simpler for you to keep this previous parameter!

Reply:
I explicitly checked with Sebastien and he told me to use CNRS/INP3, right (@sbinet )?